MAINTAINERS.yml

```diff
@@ -3131,7 +3200,7 @@ West:
     - drivers/misc/ethos_u/
     - modules/hal_ethos_u/
   labels:
-    - manifest-hal_ethos_u
+    - "area: ARM"
```

Comment:
is this right?

Reply:
suggest other labels if  have something in mind, I could not find anything else

Reply:
oh sorry, I thought it was a platform, nah sounds about right